cloud/shared/bin/lib/terraform.sh

```diff
@@ -10,22 +18,23 @@
 #######################################
 function terraform::perform_apply() {
   if [[ "${CIVIFORM_MODE}" == "dev" ]]; then
-    terraform -chdir="${TERRAFORM_TEMPLATE_DIR}" init -upgrade
+    ${TERRAFORM_BASE_COMMAND} init -upgrade
```

Comment:
Need quotes around all the new ${} commands similar to the below command you're keeping.

Reply:
ahh i thought it would make a string

Reply:
ya this doesn't work as expected-- i can switch it

Reply:
okay i think this is now what the style guide reccomends

Reply:
#sorry.....  So many characters